templates/ec2-bootstrap-trigger.yaml

```diff
@@ -0,0 +1,113 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: >-
```

Comment:
I see, so theres an ec2 triggering the AWS glue jobs?

Reply:
Yes.